website/docs/r/datadog_monitors.html.markdown

```diff
@@ -0,0 +1,146 @@
+---
+subcategory: "Datadog"
+layout: "azurerm"
+page_title: "Azure Resource Manager: azurerm_datadog_monitor"
+description: |-
+  Manages a datadog Monitor.
+---
+
+# azurerm_datadog_monitor
+
+Manages a datadog Monitor.
+
```

Comment:
I wasn't able to get this working with a service principal. Were you? If not, we should call out that you need to use cli auth to manage this resource

```suggestion
!> **Authentication** The API for this resource does not support service principal authentication. This resource can only be used with Azure CLI authentication.
```

Reply:
As mentioned above, we are only deploying one flow that supports SPN login.
Is it recommended to have flows where SPN login is not supported?

Reply:
Virtually every other resource in the provider supports provisioning via a Service Principal, is that API limitation planned to be fixed?

Reply:
Yes
We plan to fix this limitation for new resource creation in future.

Reply:
Any idea when that's likely to be fixed? From our side we'd need that to be able to run the tests, so it's a prerequisite to supporting this unfortunately

Reply:
Linking scenario is supported via APN authentication.

Reply:
APN?

Reply:
My bad. SPN*